src/features/atomics.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Atomics.wait, Atomics.notify, Atomics.waitAsync'
+author: '[Marja Hölttä](https://twitter.com/marjakh), a non-blocking blogger'
+avatars:
+  - marja-holtta
+date: 2020-09-14
+tags:
+  - ECMAScript
+  - ES2020
+description: 'Using Atomics to implement a mutex with synchronous and asynchronous modes.'
```

Comment:
I wonder if this should be a short description of the feature itself, to make clear this is a feature explainer page, rather than a blog post about implementing a mutex.

Reply:
ok, changed that, but where is this text even visible?